builder/vmware/common/step_configure_vmx.go

```diff
@@ -144,56 +144,56 @@ func (s *StepConfigureVMX) Run(ctx context.Context, state multistep.StateBag) mu
 }
 
 type DiskAndCDConfigData struct {
-	SCSI_Present         string
-	SCSI_diskAdapterType string
-	SATA_Present         string
-	NVME_Present         string
-
-	DiskType                   string
-	CDROMType                  string
-	CDROMType_PrimarySecondary string
-	CDROM_PATH                 string
+	ScsiPresent         string
+	ScsiDiskadaptertype string
```

Comment:
Nit but shouldn't this be ScsiDiskAdapterType, to match the pascal casing and snake casing used otherwise in the plugin, I have the same thought for CdromtypePrimarysecondary.

Reply:
Will request a review once it's ready, so we can park it until you return. 😀